.travis.yml

```diff
@@ -1,12 +1,23 @@
-language: node_js
-node_js:
-  - lts/*
-  - 14
-  - 13
-  - 12
-  - 11
-  - 10
-script:
-  - npm run lint
-  - npm test
-  - npm run coverage
+matrix:
+  include:
+    - language: node_js
+      node_js:
+        - lts/*
+        - 14
+        - 13
+        - 12
+        - 11
+        - 10
```

Comment:
Looks like Travis takes the first item in this array and ignores the rest, so it only runs the build for `lts/*`.

See the generated JSON build config from [here](https://travis-ci.com/github/balazsbotond/urlcat/builds/187936230/config):

```json
{
  "language": "ruby",
  "os": [
    "linux"
  ],
  "dist": "xenial",
  "jobs": {
    "include": [
      {
        "language": "node_js",
        "node_js": "lts/*",
        "script": [
          "npm run lint",
          "npm test",
          "npm run coverage"
        ]
      },
      {
        "language": "shell",
        "services": [
          "docker"
        ],
        "script": [
          "docker build -f Dockerfile.deno -t urlcat-deno .",
          "docker run --rm urlcat-deno run ./src/index.ts",
          "docker run --rm urlcat-deno bundle ./src/index.ts",
          "docker run --rm urlcat-deno lint --unstable src/",
          "docker rmi urlcat-deno"
        ]
      }
    ]
  }
}
```

I think the `language: node_js` section needs to be repeated for each node version.